handlers/main.yml

```diff
@@ -4,16 +4,18 @@
   command: apache2ctl configtest
   notify: [ 'Reload apache' ]
 
-- name: Test apache and restart
-  command: apache2ctl configtest
-  notify: [ 'Restart apache' ]
+# - name: Test apache and restart
+#   command: apache2ctl configtest
+#   notify: [ 'Restart apache' ]
```

Comment:
Forgotten debugging? I would leave unused handlers uncommented, they shouldn't affect the role. Commented out code looks weird, IMO.